include/swift/Driver/ToolChain.h

```diff
@@ -205,7 +205,7 @@ class ToolChain {
 
   /// Get the resource dir link path, which is platform-specific and found
   /// relative to the compiler.
-  void getResourceDirPath(SmallVectorImpl<char> &runtimeLibPath,
+  void getResourceDirPath(SmallString<128> &runtimeLibPath,
```

Comment:
Why this change?  The `SmallVectorImpl` is the stack allocation size erased base type for `SmallString`.  This allows for different sizes to be used.

Reply:
reverted